### PR TITLE
[docs] New homepage; reorg docs; add start of eng design docs

### PR DESCRIPTION
**Pull request by advayDev1**
* Add skeleton of eng design docs
* Make home page describe where to find info
* Make all markdown pretty

### CODE REVIEW

File: server/root [hunk 1]
```diff
@@ -1 +0,0 @@
-../
```
Comment (epicfaace) [inline server/root:1]: Why do we no longer need this file?

Comment (advayDev1) [reply server/root:1]: It was causing problems with globs.  `**/*.md` from the root would go into an infinite loop.
Tests are passing without the symlink, let me see if anything else is broken

File: docs/ONBOARDING.md [hunk 2]
```diff
@@ -1,50 +1,52 @@
...
 
-* GitHub Issues: https://github.com/WorldHealthOrganization/app/issues
-* Slack: _We apologize, but the Slack is closed to new contributors until initial app release_
-* WIP product roadmap: [ROADMAP.md](ROADMAP.md)
-* Rough engineering backlog: https://github.com/WorldHealthOrganization/app/projects/1
-* Most up-to-date preview of app in development: _Will be re-added soon_
+- GitHub Issues: https://github.com/WorldHealthOrganization/app/issues
+- Slack: _We apologize, but the Slack is closed to new contributors until initial app release_
+- Rough engineering backlog: https://github.com/WorldHealthOrganization/app/projects/1
+- Most up-to-date preview of app in development: _Will be re-added soon_
```
Comment (hspinks) [inline docs/ONBOARDING.md:16]: Remove

```suggestion

```

File: docs/README.md [hunk 3]
```diff
@@ -4,9 +4,16 @@
...
+| There's a problem with the app - help!                   | File a report to help us improve. Choose "Bug Report" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (hspinks) [inline docs/README.md:15]: Link directly to Bug Report template?

```suggestion
| There's a problem with the app - help!                   | File a new bug report [here](https://github.com/WorldHealthOrganization/app/issues/new?template=bug-report.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

File: docs/README.md [hunk 4]
```diff
@@ -4,9 +4,16 @@
...
+| I have an idea for the app!                              | Choose "Feature Request" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (hspinks) [inline docs/README.md:16]: Link directly to Feature Request template?

```suggestion
| I have an idea for the app!                              | Request a feature [here](https://github.com/WorldHealthOrganization/app/issues/new?template=feature-request.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

File: docs/README.md [hunk 5]
```diff
@@ -4,9 +4,16 @@
...
+| Who can I talk to about product, engineering, or design? | Reach out to our [functional leads](ONBOARDING.md#point-people).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (hspinks) [inline docs/README.md:18]: I'm fine with this, but we are signing ourselves up for a lot of emails to filter through in this case... should we work with PgMs to think through a better place to direct people? Google Group maybe?

File: docs/devdesign/subteams.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+TODO: All other parts of this document past v1.
+
+For v1, we will work in the following subteams:
+
+- Client UI - @patniemeyer , @theswerd
+- Localization and internationalization - @sapte91 , @clementmouchet
+- Instrumentation/analytics - @epicfaace
+- Push notifications - ChrisG, @patniemeyer
```
Comment (hspinks) [inline docs/devdesign/subteams.md:8]:
```suggestion
- Push notifications - @cbgrey, @patniemeyer
```

File: docs/devdesign/subteams.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+TODO: All other parts of this document past v1.
```
Comment (hspinks) [inline docs/devdesign/subteams.md:1]: Title for consistency:

```suggestion
## Subteams

TODO: All other parts of this document past v1.
```

File: docs/labels.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+We use labels liberally for our issues and pull requests.
```
Comment (hspinks) [inline docs/labels.md:1]: Title for consistency:

```suggestion
## GitHub Labels

We use labels liberally for our issues and pull requests.
```

File: docs/README.md (hunk 3 above)
Comment (advayDev1) [reply docs/README.md:15]: We can't do that.  The link doesn't just have the bug report template it, it includes the labels and other content which change over time.  No idea why GitHub does it that way :(

File: docs/README.md (hunk 4 above)
Comment (advayDev1) [reply docs/README.md:16]: We can't do that.  The link doesn't just have the bug report template it, it includes the labels and other content which change over time.  No idea why GitHub does it that way :(

File: docs/README.md (hunk 5 above)
Comment (advayDev1) [reply docs/README.md:18]: I think we should do that, but at this moment we don't have that process.  Once we have that process, we can change this.

**Comment by advayDev1** [comment]
ptal @hspinks

**Pull request merged**